code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -493,6 +493,23 @@
 	..()
 	. = 1
 
+/datum/reagent/medicine/pen_jelly
```

Comment:
Give the `/datum/reagent/medicine/pen_acid` type a `var/heal_slimes` (example name) defaulting to FALSE, then add `heal_slimes` as third arg on line 489.
Then just make this a subtype of pentetic acid with the heal_slimes var set to TRUE and remove redunancy.

Reply:
Learn something new every day. Took your suggestion, thank you.